### PR TITLE
reporter-web-app: Go back to using Yarn 1.9.4 again

### PR DESCRIPTION
**Pull request by sschuberth**
Partly revert 5e4ab2c to work around an issue on AppVeyor. While it was
nice to align with the analyzer on version 1.12.1, it is not strictly
required for this project.

Signed-off-by: Sebastian Schuberth <sebastian.schuberth@here.com>

---
This change is [Reviewable](https://reviewable.io/reviews/heremaps/oss-review-toolkit/1018)

### CODE REVIEW

**Comment by sschuberth** [comment]
Added a few more commits in the hope to stabilize AppVeyor checks.

File: scanner/src/funTest/kotlin/HttpCacheTest.kt [hunk 1]
```diff
@@ -76,15 +76,15 @@ class HttpCacheTest : StringSpec() {
...
         server = HttpServer.create(InetSocketAddress(loopback, port), 0)
         server.createContext("/", MyHttpHandler())
         server.start()
     }
 
     override fun afterTest(description: Description, result: TestResult) {
-        // Ensure the server is properly stopped even in case of exceptions.
-        server.stop(0)
+        // Ensure the server is properly stopped even in case of exceptions, but wait at most 5 seconds.
+        server.stop(5)
```
Comment (fviernau) [inline scanner/src/funTest/kotlin/HttpCacheTest.kt]: Is it actually necessary to start / stop the server before each test cases.
I'm not sure how heavy weigth this is I have to admit - if it is slow would it be sufficient to 
override beforeClass / afterClass instead?

Comment (sschuberth) [reply scanner/src/funTest/kotlin/HttpCacheTest.kt]: We should definitely do this for cleanliness. But I'm also going to further change the test to allow test to run in parallel.

Comment (sschuberth) [reply scanner/src/funTest/kotlin/HttpCacheTest.kt]: Turns out kotlintest always [runs test cases in a test spec sequentially](https://github.com/kotlintest/kotlintest/blob/master/doc/reference.md#parallelism), so no need for a change here.

Comment (fviernau) [reply scanner/src/funTest/kotlin/HttpCacheTest.kt]: Sorry, my comment was quite off topic. Let's forget about this for now.

**Pull request merged**